### DISCUSSION TITLE
Feature Request: Direction and Unicode utilities for bidirectional and multilingual websites

### DISCUSSION THREAD

**Discussion by KL13NT** [tailwindlabs/tailwindcss#2674] [Ideas]
I'm building a personal blog that has posts written in both Arabic (RTL) and English, and based on a language key that gets passed to specific components I determine whether such components should be rendered RTL or LTR. I searched through the tailwind documentation for `unicode-bidi` and `direction` but didn't find them. If they're not available in Tailwind yet, is such a feature added to some backlog or perhaps would be added? I'm willing to work on this myself if approved.

**Comment by adamwathan** [comment]
Hey! Our recommendation has always been to use a plugin like this one:

https://github.com/RonMelkhior/tailwindcss-dir

It might be something we add officially one day but it's not on the roadmap right now (we only plan ahead about 6-12 weeks at a time and don't keep a backlog beyond that).

**Reply by KL13NT** [reply]
Oh, that's neat! I'll make sure to check it out. Thanks for the quick reply!
